gflownet/utils/logger.py

```diff
@@ -71,14 +71,21 @@ def __init__(
         self.lightweight = lightweight
         self.debug = debug
         # Log directory
-        self.logdir = Path(logdir.root)
+        if "path" in logdir:
+            self.logdir = Path(logdir.path)
+        else:
+            self.logdir = Path(logdir.root)
```

Comment:
Why is this needed? It seems very arbitrary.

Reply:
Not sure if it's strictly needed but it's additional flexibility. Currently, `logdir.path` is created in `main.py`, but this will make it still work if the code is run from a different script that does not do that.